client/src/scenes/SettingsProfile/index.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import {
+    Grid,
+    Menu,
+    Header,
+    Divider,
+    Image,
+    Input,
+    Button,
+    TextArea,
+    Icon,
+    Segment,
+    Dropdown
+} from 'semantic-ui-react';
+import { Formik, Form, Field } from 'formik';
+import NOTES from './notes';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import PageHeader from '../../containers/Header';
+import { updateUserSettings } from '../../routines/routines';
+
+import styles from './styles.module.scss';
+import avatarPlaceholder from './avatar-placeholder.svg';
+
+class SettingsProfile extends Component {
+  renderField = ({ field }) => <Input fluid {...field} />;
+
+  renderDisabledField = ({ field }) => <Input disabled fluid {...field} />;
+
+  renderTextArea = ({ field }) => (
+      <TextArea {...field} rows="3" placeholder="Tell us a little bit about yourself" className={styles.textarea} />
+  );
+
+  onSubmit = (values, { setSubmitting }) => {
+      const { id, bio, company, location, name, imgUrl, url } = values;
+      const payload = { id, settings: { bio, company, location, name, imgUrl, url } };
+      this.props.updateUserSettings(payload);
+  };
+
+  onUploadPhoto = ev => {
+      console.warn('Upload photo!');
+  };
+
+  onRemovePhoto = ev => {
+      console.warn('Remove photo!');
+  };
+
+  imgEditBtn = (
+    <Button secondary compact size="tiny">
+        <Icon name="edit" />
+      Edit
+    </Button>
+  );
+
+  render() {
+      return (
+          <div>
```

Comment:
Если нету никаких стилей связанных с этим div-ом, то можно заменить
```suggestion
          <>
```